src/plugins/conventional-commits/index.ts

```diff
@@ -0,0 +1,36 @@
+import { applyPlugins, parse, plugins } from 'parse-commit-message';
+
+import { AutoRelease, IPlugin } from '../../main';
+import { VersionLabel } from '../../release';
+
+export default class ConventionalCommitsPlugin implements IPlugin {
+  name = 'Conventional Commits Parser';
+
+  apply(auto: AutoRelease) {
+    auto.hooks.onCreateLogParse.tap(this.name, logParse => {
+      logParse.hooks.parseCommit.tap(this.name, commit => {
+        try {
+          const [conventionalCommit] = applyPlugins(
+            [plugins[1]],
```

Comment:
You can use `mappers` here. It is like "named plugins". It will be more clear what plugin you include. Also, the first argument to `applyPlugins`, don't need to be an array, it's arrayified.

Reply:
Cool! Thanks!